impeller/renderer/backend/vulkan/command_pool_vk.cc

```diff
@@ -158,6 +158,15 @@ void CommandPoolVK::Destroy() {
 // Associates a resource with a thread and context.
 using CommandPoolMap =
     std::unordered_map<uint64_t, std::shared_ptr<CommandPoolVK>>;
+
+// Lifecycle:
```

Comment:
```suggestion
// CommandPoolVK Lifecycle:
```

Anywhere where you talk about a "command pool" I'd switch it to "CommandPoolVK".  We should make it clear that this is referring to the things inside of the `tls_command_pool_map`, not the `tls_command_pool_map` itself.